Libraries/Image/Image.ios.js

```diff
@@ -108,19 +108,34 @@ const ImageViewManager = NativeModules.ImageViewManager;
  * You will need to add some optional modules in `android/app/build.gradle`, depending on the needs of your app.
  *
  * ```
+ * // use getFrescoComponent to lock your components to the version used by react-native
+ * def getFrescoVersion() {
+ *    String reactGradle = file('../../node_modules/react-native/ReactAndroid/build.gradle').text
+ *    // use the default of the latest version
+ *    def version = '+'
```

Comment:
the alternative here is that it just throws an error when it can't find a correct version. doing this will likely just continue the same problem. any thoughts on best practices?